apps/chat-e2e/src/tests/chatSelectionFunctionality.test.ts

```diff
@@ -0,0 +1,251 @@
+      await conversationAssertion.assertSelectedConversation(
+        playbackConversation,
+      );
+    });
+
+    await dialTest.step('Click on Export', async () => {
+      await conversations.openEntityDropdownMenu(replayConversation);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.export);
+      const downloadedData = await dialHomePage.downloadData(() =>
+        conversations
+          .getDropdownMenu()
+          .selectMenuOption(MenuOptions.withoutAttachments),
+      );
+      expect(downloadedData.path).toBeTruthy();
+      expect(downloadedData.path).toMatch(/\.json$/);
+      await conversationAssertion.assertSelectedConversation(
+        playbackConversation,
+      );
+    });
+
+    await dialTest.step(
+      'Click on Delete while another chat is selected',
+      async () => {
+        await conversations.openEntityDropdownMenu(replayConversation);
+        await conversations
+          .getDropdownMenu()
+          .selectMenuOption(MenuOptions.delete);
+        await confirmationDialog.confirm({ triggeredHttpMethod: 'DELETE' });
+        await conversationAssertion.assertSelectedConversation(
+          playbackConversation,
+        );
+        await conversationAssertion.assertEntityState(
+          { name: replayConversation },
+          'hidden',
+        );
+        replayConversation = '';
+      },
+    );
+
+    await dialTest.step('Click on "Move to -> New folder"', async () => {
+      await conversations.openEntityDropdownMenu(secondConversation.name);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.moveTo);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.newFolder);
+      await conversationAssertion.assertSelectedConversation(
+        playbackConversation,
+      );
+      await folderConversations.expandFolder(
+        ExpectedConstants.newFolderWithIndexTitle(1),
+      );
+      await chatBarFolderAssertion.assertFolderState(
+        { name: ExpectedConstants.newFolderWithIndexTitle(1) },
+        'visible',
+      );
+      await chatBarFolderAssertion.assertFolderEntityState(
+        { name: ExpectedConstants.newFolderWithIndexTitle(1) },
+        { name: secondConversation.name },
+        'visible',
+      );
+      secondConversation.folderId = `${secondConversation.folderId}/${ExpectedConstants.newFolderWithIndexTitle(1)}`;
+    });
+
+    await dialTest.step('Click on Duplicate', async () => {
+      await conversations.openEntityDropdownMenu(firstConversation.name, 2);
+      await conversations
+        .getDropdownMenu()
+        .selectMenuOption(MenuOptions.duplicate);
+      clonedConversation = `${firstConversation.name} 1`;
+      await conversationAssertion.assertSelectedConversation(
+        clonedConversation,
+      );
+    });
+
+    await dialTest.step('Click on Share', async () => {
```

Comment:
this is automated in scope of sharedChatIcons.test.ts tests, can be removed

Reply:
Updated